community-packages/package-list.json

```diff
@@ -271,6 +271,10 @@
     {
       "repoSlug": "pulumiverse/pulumi-talos",
       "schemaFile": "provider/cmd/pulumi-resource-talos/schema.json"
+    },
+    {
+      "repoSlug": "lbrlabs/pulumi-tailscale-bastion",
+      "schemaFile": "provider/cmd/pulumi-resource-tailscale-bastion/schema.json"
```

Comment:
@jaxxstorm The schema file is not in this location. Can you add it?

Reply:
Fixed